Recurly/BaseClient.cs

```diff
@@ -19,20 +19,21 @@ public class BaseClient
     {
         private string ApiKey { get; }
         private const string ApiUrl = "https://v3.recurly.com/";
+        private const string EuApiUrl = "https://v3.eu.recurly.com/";
         private string[] BinaryTypes = { "application/pdf" };
         private List<IEventHandler> EventHandlers = new List<IEventHandler>();
         public virtual string ApiVersion { get; protected set; }
 
         internal IRestClient RestClient { get; set; }
 
-        public BaseClient(string apiKey)
+        public BaseClient(string apiKey, string region = "us")
```

Comment:
Instead of having these hard coded strings (`"us"` and `"eu"`), lets add an enum for these:

```csharp
public enum Region
{
    US,
    EU
}
```